### PR TITLE
Object relations sample

### PR DESCRIPTION
**Pull request by desistefanova**

### CODE REVIEW

File: object_relations/bin/object_relations.dart [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import 'package:realm_dart/realm.dart';
+
+import 'model.dart';
+
+void main(List<String> arguments) {
```
Comment (desistefanova) [inline object_relations/bin/object_relations.dart:5]: Main doesn't exits and never complete when using 0.2.0-alpha. It woks perfect when dll is build from master. Are they different versions or it is something else? @blagoev @nielsenko

Comment (nielsenko) [reply object_relations/bin/object_relations.dart:5]: There is a bug in 0.2.0-alpha regarding closing realms, that has been fixed in master. Be aware that it is important to close the realm, it does not happen automatically.

File: object_relations/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: object_relations
+description: A simple command-line application.
+version: 1.0.0
+# homepage: https://www.example.com
+
+environment:
+  sdk: '>=2.16.0-134.1.beta <3.0.0'
```
Comment (nielsenko) [inline object_relations/pubspec.yaml:7]: I think you should loosen this constraint to >= 2.15.0, so the example will work with the current stable

**Review by nielsenko — CHANGES_REQUESTED** [review]

**Comment by blagoev** [comment]
I have updated the realm_dart sample with references. I think we should close this one.

**Pull request closed without merging**